### DISCUSSION TITLE
Batteries version

### DISCUSSION THREAD

**Discussion by Jymms** [ericsink/SQLitePCL.raw#589] [Q&A]
Curious using SQLitePCLRaw.bundle_e_sqlcipher (2.1.8) Which call is correct?
 SQLitePCL.Batteries.Init();
or
SQLitePCL.Batteries_V2.Init();

Thanks

**Comment by ericsink** [comment]
It doesn't matter.  A very long time ago, these calls were different, but the reason for that disappeared.  They're identical, and I keep them both around simply to preserve backward compatibility.
